### DISCUSSION TITLE
Can only open one .mm from OneDrive folders

### DISCUSSION THREAD

**Discussion by PingPongTechnologies** [freeplane/freeplane#1418] [Help]
I just copied two files from my OneDrive folder to a 'shared' OneDrive folder. I can open one of them from the same location or two from two different locations but NOT two files from the same location.

These were the first two I've made with Freeplane, so I made some quick trash ones to test with the same results.

I found a comment about Google Drive, but I don't think it's the same issue. https://github.com/freeplane/freeplane/discussions/1000

Edit: When I put 3 .mm files on the desktop, they all opened without any problems.

**Comment by theelectricsoul** [comment]
I do not have any issues when I edit mm files from within a shared onedrive folder. Mabye I do not understand what you do. I know that I am using a shared folder, because I can see the respective icon.

![image](https://github.com/freeplane/freeplane/assets/10260832/0e95972c-2afa-4732-a3f2-b4919555d948)

**Reply by PingPongTechnologies** [reply]
I can open and edit one at a time from the same OneDrive folder, but I can't open more than one from the same folder. Instead of opening the second file, it just makes Freeplane blink on the taskbar. Thank you for giving it a look!

**Reply by theelectricsoul** [reply]
try another machine ? :)
